spec/setup_acceptance_node.pp

```diff
@@ -23,26 +23,14 @@
   default: {}
 }
 
-# Defaults to staging, for release, use
-# $baseurl = "https://fedorapeople.org/groups/katello/releases/yum/nightly/pulpcore/el${major}/x86_64/"
-$baseurl = "http://koji.katello.org/releases/yum/katello-nightly/pulpcore/el${major}/x86_64/"
-
-$pulpcore_repo_conf = @("CONF")
-[pulpcore]
-baseurl=${baseurl}
-gpgcheck=0
-module_hotfixes=1
-CONF
-
-file { '/etc/yum.repos.d/pulpcore.repo':
-  ensure  => file,
-  content => $pulpcore_repo_conf,
+class { 'pulpcore::repo':
+  version => fact('pulpcore_version'),
```

Comment:
Once https://github.com/voxpupuli/voxpupuli-acceptance/pull/13 is merged and released in the gem, you can use `BEAKER_FACTER_PULPCORE_VERSION=3.7` when running beaker to test out version 3.7. If it's undefined, it falls back to the class default (3.6).